_layouts/page.html

```diff
@@ -54,13 +51,18 @@ <h1 class="post-title-main display-2">{{ page.title }}</h1>
 
     <div class="tags">
         {% if page.tags != null %}
-	        Tags<br />
+            {% if page.tags > 0 %}
```

Comment:
This logic can show the "Tags" label when no tags are visible. :-/

If a page has 1 tag, `page.tags = ['banana']`, and `site.data.tags.allowed-tags` doesn't include `banana`, then "Tags" will display with no tags visible.

Reply:
It doesn't look like any of the tutorial pages (that use `_layouts/tutorial.html`) have tags and every page that isn't a tutorial (that use `_layouts/page.html`) has tags. Because they're two different templates, it's a little harder to test. :/